src/bin/check-icu4c-version.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+if [ $# -ne 0 ];
+then
+    cd $1
+    shift
+fi
+
+if [ ! -f  ./source/common/unicode/uversion.h ];
+then
+    echo "unknown-no-uversion"
+    exit 1
+fi
+( grep -h "^#define U_ICU_VERSION " ./source/common/unicode/uvernum.h || echo unknown-no-define) | cut -d' ' -f3 | tr -d '"'
```

Comment:
The `-h` option is the default when `grep` only has one file to search, it's redundant here.

Reply:
I originally had included `source/common/unicode/uversion.h`, however, it was not needed. (it predates 4.8)